sql/sql_acl.cc

```diff
@@ -903,7 +903,7 @@ class User_table: public Grant_table_base
   virtual int get_auth(THD *, MEM_ROOT *, ACL_USER *u) const= 0;
   virtual bool set_auth(const ACL_USER &u) const = 0;
   virtual privilege_t get_access() const = 0;
-  virtual void set_access(const privilege_t rights, bool revoke) const = 0;
+  virtual void set_access(const privilege_t rights) const = 0;
```

Comment:
No, I don't agree with this one. It wasn't hiding anything, it was **set**ting values of specific privilege bits or columns to the value specified by the second argument. It was a quite clear and well defined semantics. And `set()` already was idempotent, as far as I understand.